[RFE] Provide "all applications" to rules creation

On creating a new rule (for example for "My Events" or "My artifacts") I would like to select "all applications", so in future when new apps are added I don't have to remember to update my rule.

npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /home/ruth/Fedora/fmn/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/home/ruth/Fedora/fmn/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in: /home/ruth/.npm/_logs/2024-03-19T16_07_23_694Z-debug-0.log

That is the error

--- a/frontend/src/components/rule-edit/generation-rule/BaseFilterApplication.vue
+++ b/frontend/src/components/rule-edit/generation-rule/BaseFilterApplication.vue
@@ -11,6 +11,7 @@ const props = defineProps<{
   name: string;
   label: string;
   placeholder?: string;
+  help?: string;
 }>();
 
 const url = "/api/v1/applications";
@@ -33,5 +34,6 @@ const getApplications = async () => {
     :msOptions="getApplications"
     searchable
     :close-on-select="false"
+    :help="props.help"
   />
 </template>

--- a/frontend/src/components/rule-edit/generation-rule/FilterApplication.vue
+++ b/frontend/src/components/rule-edit/generation-rule/FilterApplication.vue
@@ -13,5 +13,6 @@ import BaseFilterApplication from "./BaseFilterApplication.vue";
     name="applications"
     label="Applications"
     placeholder="Choose applications"
+    help="If you leave this field empty, the rule applies to all applications (including new ones added later)"
   />
 </template>

--- a/frontend/src/forms/index.ts
+++ b/frontend/src/forms/index.ts
@@ -53,6 +53,12 @@ export const config = defaultConfig({
         input: "form-check-input",
         help: "text-muted",
       },
+      multiselect: {
+        help: "text-muted",
+      },
+      multiselectasyncdefault: {
+        help: "text-muted",
+      },
     }),
   },
 });

